FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/FreeRTOS_ARP.c

```diff
@@ -88,6 +88,10 @@ static TickType_t xLastGratuitousARPTime = ( TickType_t ) 0;
 /* Part of the Ethernet and ARP headers are always constant when sending an IPv4
 ARP packet.  This array defines the constant parts, allowing this part of the
 packet to be filled in using a simple memcpy() instead of individual writes. */
+
+/* Rule has been relaxed since this is a configuration parameter. And this location provides
+ * more readability. */
```

Comment:
Assuming we are done with functional review, planning to glance over cosmetics. 

Minor: please keep comment block style consistent. This is not a blocker. 
```
/* line 1
line 2
line 3
*/
```
or 
```
/* line 1
 * line 2
 * line 3
 */
```

And, have a single block. So preferably _not_ 
```
statements...
/* don't */
/* do */
/* this */
statements...
```

```
statement...
/* Yes,
this looks better. */
statement...
```

Reply:
Since we have multiple PRs just for Coverity, for all the styling issues we could do a PR for entire +TCP after everything is in. And if that's the rule we all agree on (please check with all stakeholders), suggesting to leave out all style changes to make review easier. 

I'm not saying style is not important. I think fixing Coverity violations is more crucial. And also suggesting to get CI team involved to setup style checks.

Reply:
With the latest PR, I added consistent comment style.
I am unsure about writting a big block of comment in case I am trying to suppress Coverity.
I believe I should write /* coverity[misra_related_warning_violation] */ to suppress the warnings.

Reply:
About coding style: all FreeRTOS code used to have this style:

~~~c
/* line 1
line 2
line 3 */
~~~

Currently, I see that this style is also used in `tasks.c`:

~~~c
/*
 * line 1
 * line 2
 * line 3
 */
~~~
So things might be changing...

Personally I like the second style

What is important is to start a comment with a capital letter, and end it with a period.

Reply:
I am using the second style in +TCP since the first block comment in every file (the license) is of that format.